home/components/ProjectView.tsx

```diff
@@ -164,10 +173,11 @@ function LegacyLaunchSection({ app }: { app: ProjectDataProject }) {
         }}
         last
       />
-      <Text style={styles.moreLegacyBranchesText}>
-        To launch a update from another classic updates release channel, scan the QR code on project
-        webpage.
-      </Text>
+      {Platform.OS === 'ios' ? null : (
```

Comment:
Probably don't need this platform fork since we updated the copy.

Reply:
i'd like to avoid mentioning the qr code on ios, blemflarck has disliked this in the past and specifically called it out